dashboard/config/scripts/levels/Jigsaw_Level_1.level

```diff
@@ -0,0 +1,16 @@
+<Blockly>
+  <config><![CDATA[{
+  "game_id": 27,
+  "created_at": "2014-09-20T02:48:46.000Z",
+  "level_num": "1",
+  "properties": {
+    "skin": "jigsaw",
+    "preload_asset_list": null
+  },
+  "published": false,
+  "level_concept_difficulty": {
+    "sequencing": 1
+  }
+}]]></config>
+  <blocks/>
+</Blockly>
```

Comment:
is it expected that this is an entirely new level?

Reply:
i think so, it looks like this level was created in 2014 and never had a .level file

Reply:
[follow-up to our in-person chat] this level is a javascript-only level, so it _should not_ have a .level file. since i horribly timed this and merged the PR right before the content scoop, i fixed this manually by:

1. delete this file (`Jigsaw_Level_1.level`) on levelbuilder
2. commit/push to the levelbuilder branch, which updated the levelbuilder-to-staging PR
3. merged the levelbuilder content PR

thankfully, this never made it to staging because there was a merge conflict in the lb-to-staging PR. i'll update the one-off script to make sure this doesn't happen in the future!